python/beeai_framework/context.py

```diff
@@ -194,6 +196,7 @@ def _on_abort() -> None:
             finally:
                 await emitter.emit("finish", None)
                 context.destroy()
+                emitter.destroy()
```

Comment:
this is redundant, `context.destroy()` calls `emitter.destroy()`

Reply:
Not really. This is a child instance.

Reply:
Ah, I see, I had thought that `emitter.destroy()` also ran `emitter.destroy()` on it's children but it seems I was wrong